src/index.html

```diff
@@ -15,8 +15,101 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <!-- <link rel="stylesheet" href="https://fonts.google.com/specimen/Roboto"> -->
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          class="logo__image"
+          alt="Moyo Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a
+              href="#"
+              class="nav__item is-active"
+            >
```

Comment:
Remember about correct indentation between parent and child elements. The `<a>` tag starting on line 47 should be indented as it is a child of the `<li>` tag.